### PR TITLE
Add jc_custom_fields to Cb::Models::Job

### PR DESCRIPTION
**Pull request by xwang-cb**
Add jc_custom_fields to Cb::Models::Job
--APACLocal team

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@ Version History
 ====
     * All Version bumps are required to update this file as well!!
 ----
+* 21.3.6 Add jc_custom_fields to Cb::Models::Job
```
Comment (oochelz) [inline CHANGELOG.md]: For a new feature, I would make this a minor version bump (21.4.0).

File: lib/cb/models/implementations/job.rb [hunk 2]
```diff
@@ -181,6 +182,9 @@ def set_model_properties
         @zip                          = args['LocationPostalCode'] || ''
         @posting_date                 = args['PostingDate'] || ''
 
+        # JC Custom Fields related
+        @jc_custom_fields             = args['JCCustomField'] || ''
```
Comment (oochelz) [inline lib/cb/models/implementations/job.rb]: Please remove the comment (this is self-explanatory), and add a unit test.

Comment (xwang-cb) [inline lib/cb/models/implementations/job.rb]: Could you please give me a sample of this unit test ? I searched before, but found nothing for this case. Thanks so much

File: CHANGELOG.md (hunk 1 above)
Comment (xwang-cb) [inline CHANGELOG.md]: Done as your request.

**Comment by oochelz** [comment]
Here are example tests:
https://github.com/careerbuilder/ruby-cb-api/blob/master/spec/cb/models/implementations/job_spec.rb#L26..L41

**Comment by xwang-cb** [comment]
Thanks @oochelz, I have added unit test.

**Comment by oochelz** [comment]
Very nice. Thank you!

**Pull request merged**